packages/story-utils/README.md

```diff
@@ -70,3 +70,17 @@ test('my button 2', () => {
   expect(screen.getByText('coffee!')).toBeTruthy();
 });
 ```
+
+### generateSnapshot
+
+Auto generates snapshots for stories
+
+```js
+
+import * as storyFileExports from "./AlongLogo.stories";
+import { generateSnapshot } from "@chanzuckerberg/story-utils";
+
+test("my button", () => {
+  generateSnapshot(storyFileExports);
+});
```

Comment:
Thanks for adding an example! I think we'll want this to show a `describe` instead of a `test`, since `generateSnapshot` writes `test`s.

```suggestion
describe("my button", () => {
  generateSnapshot(storyFileExports);
});
```